MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include LICENSE
+include README.md
+include requirements.txt
```

Comment:
Can you be clearer about your objection? Why object to `README.md` but not to `LICENSE`? I'm new to `MANIFEST.in`s, but I thought you would want to include these two files with your distribution so that the end-user has easy access to them.